src/ports/postgres/modules/deep_learning/madlib_keras.py_in

```diff
@@ -198,11 +199,11 @@ def fit(schema_madlib, source_table, model, dependent_varname,
         final_validation_loss = validation_aggregate_loss[-1]
     version = madlib_version(schema_madlib)
     class_values, class_values_type = get_col_value_and_type(
-        fit_validator.source_summary_table, CLASS_VALUES_COLNAME)
+        fit_validator.source_summary_table, CLASS_VALUES_CNAME)
```

Comment:
Why CNAME instead of COLNAME ? I would prefer COLNAME.

Reply:
Variables for some column names were getting very big, hence `CNAME`. I don't feel strongly between CNAME or COLNAME, so have changed it to COLNAME, and it's just a couple of chars anyway, so why not! :)